libs/ui-lib/lib/ocm/components/AddHosts/day2Wizard/Day2ClusterDetails.tsx

```diff
@@ -246,10 +247,11 @@ const Day2ClusterDetails = () => {
                     </Alert>
                   </GridItem>
                 )}
-
-                <GridItem>
-                  <Day2HostStaticIpConfigurations />
-                </GridItem>
+                {day2Cluster.platform?.type !== 'oci' && (
+                  <GridItem>
+                    <Day2HostStaticIpConfigurations />
+                  </GridItem>
+                )}
```

Comment:
For consistency's sake, I think we should disable the controls with a tooltip - not hide them.

Reply:
added comment in ticket. Waiting a response